README.md

```diff
@@ -11,7 +11,7 @@ We highly recommend using [Visual Studio Code](https://code.visualstudio.com/) f
 ## Team Members
 - Anthony Weinko
 - Daniel Khoo
-- Madelyn Dubuk
+-[Madelyn Dubuk](https://github.com/madelyndubuk) 
```

Comment:
```suggestion
- [Madelyn Dubuk](https://github.com/madelyndubuk) 
```
You forgot a space between the bracket and the `-` sign, so the formatting's messed up